src/main/java/org/carlmontrobotics/Constants.java

```diff
@@ -371,32 +371,39 @@ public static final class Autoc {
 					2 * Math.PI); // The constraints for this path. If using a differential drivetrain, the
 									// angular constraints have no effect.
 		}
-		// #endregion
 	}
 
 	public static final class Limelightc {
 		public static final String INTAKE_LL_NAME = "limelight-intake";
 		public static final String SHOOTER_LL_NAME = "limelight-shooter";
 
-		public static final double ERROR_TOLERANCE_RAD = 0.1;
-		public static final double HORIZONTAL_FOV_DEG = 0;
-		public static final double RESOLUTION_WIDTH_PIX = 640;
-		public static final double MOUNT_ANGLE_DEG_SHOOTER = 42.5; 
-		public static final double MOUNT_ANGLE_DEG_INTAKE = -22; // 23.228
-		public static final double HEIGHT_FROM_GROUND_METERS_SHOOTER = Units.inchesToMeters(11.5); // 16.6
-		public static final double HEIGHT_FROM_GROUND_METERS_INTAKE = Units.inchesToMeters(52); // 16.6
-		public static final double ARM_TO_OUTTAKE_OFFSET_DEG = 115;
-		public static final double NOTE_HEIGHT = Units.inchesToMeters(0);
+		public static final int[] VALID_IDS = { 4, 7 };
```

Comment:
Why are the only valid ids used for pose estimation just these two? You should probably use all the apriltags since the more apriltags it detects, the more accurate the readings will be.

Reply:
See this [link](https://docs.limelightvision.io/docs/docs-limelight/pipeline-apriltag/apriltag-robot-localization) for more details

Reply:
you sent a link for MegaTag, but in the code I implement MegaTag2. according to [this](https://docs.limelightvision.io/docs/docs-limelight/pipeline-apriltag/apriltag-robot-localization-megatag2), MegaTag2 allows for tags to be filtered out because one is precise, and this makes it easier to focus on only the necessay tags

Reply:
Whoops my bad. Comment resolved